src/proteus/dswx_hls.py

```diff
@@ -1693,49 +1694,51 @@ def _compute_diagnostic_tests(blue, green, red, nir, swir1, swir2,
     return diagnostic_layer
 
 
-def _compute_and_apply_cloud_layer(wtr_2_layer, fmask,
-                                  mask_adjacent_to_cloud_mode):
-    """Compute cloud/cloud-shadow mask and filter interpreted water layer
+def _compute_preliminary_cloud_layer(fmask, mask_adjacent_to_cloud_mode):
+    """Compute a preliminary CLOUD layer without the snow/ice
+       class. The CLOUD layer will be completed by a subsequent function
+       _add_snow_to_cloud_layer_and_apply_cloud_masking()
+       after the WTR-2 layer is generated.
 
        Parameters
        ----------
-       wtr_2_layer: numpy.ndarray
-              Cloud-unmasked interpreted water layer
        fmask: numpy ndarray
-              HLS Fmask
+            HLS Fmask
        mask_adjacent_to_cloud_mode: str
-              Define how areas adjacent to cloud/cloud-shadow should be handled.
-              Options: "mask", "ignore", and "cover"
+            Define how areas adjacent to cloud/cloud-shadow should be handled.
+            Options: "mask" - mask out these areas marking them as cloud
+            shadow (default); "ignore" - ignore the adjacent to cloud/
+            cloud shadow classification; and 'cover' - covers these areas
+            with a dilation algorithm (this option will be handled in
+            the subsequent function
+            _add_snow_to_cloud_layer_and_apply_cloud_masking())
 
        Returns
        -------
-       cloud_layer : numpy.ndarray
-              Cloud mask
-       wtr_layer : numpy.ndarray
-              Cloud-masked interpreted water layer
+       preliminary_cloud_layer : numpy.ndarray
+            Preliminary cloud mask (without the snow/ice class)
```

Comment:
Could the docstring please be updated to something like:

```suggestion
            Preliminary cloud mask (without the snow/ice class)
            with dtype of uint8 and values assigned as follows:
                0: Not masked
                1: Cloud shadow or adjacent to cloud/cloud shadow
                4: Cloud
                5: Cloud and class 1 (cloud shadow or adjacent to cloud/cloud shadow)
                255: Fill value (no data)

       See Also
       ---------
       _add_snow_to_cloud_layer : Add the snow bit encodings from `Fmask` to `preliminary_cloud_layer`
```
numpy docstring reference: [link](https://numpydoc.readthedocs.io/en/latest/format.html#see-also)

Note: the values I wrote above might be incorrect. But, this is why the docstring will be very helpful. Thanks!

Reply:
Just update the docstrings. Thank you, @nemo794

Reply:
Thank you for filling out the docstring. I'm not sure if something got mangled in the git merge, but now the output variable name and type disappeared, the indentation is off, and there should be an extra newline between the `Return` block and the `See Also` block. Could you please look into this? Thank you!

Reply:
Your recent commit just fixed the spacing in the `_add_snow_to_cloud_layer` docstring (thank you!), but the spacing issues mentioned are still present in the `_compute_preliminary_cloud_layer` docstring. Could you please update? Thank you!